playground/src/main/kotlin/kotlinx/io/bytes/ByteArrayOutput.kt

```diff
@@ -0,0 +1,32 @@
+package kotlinx.io.bytes
+
+import kotlinx.io.*
+import kotlinx.io.buffer.*
+
+class ByteArrayOutput(initialSize: Int = 16 ) : Output() {
+    init {
+        require(initialSize > 0)
+    }
+    private var array: ByteArray = ByteArray(initialSize)
+    private var size = 0
+
+    public fun toArray(): ByteArray = array.copyOf(size)
+
+    override fun flush(source: Buffer, length: Int) {
+        ensureCapacity(length)
+        for (i in 0 until length) {
```

Comment:
Please consider the `repeat` function

Reply:
This module is named `playground` to emphasize its (much) lower requirements for code quality.
It's neither production-ready nor feature-complete. 
We have no interest in bikeshedding implementation details, naming or code-style of this module